.gitmodules

```diff
@@ -59,7 +59,7 @@
 [submodule "third_party/libunwind"]
 	path = third_party/libunwind
 	url = https://github.com/tarantool/libunwind.git
-	branch = libunwind-1.6.2-tarantool
+	branch = libunwind-f67ef28-tarantool
```

Comment:
Why do you need to specify branch here?

Anyway, IMO better just cherry-pick the required commit to `libunwind-1.6.2-tarantool` instead of adding a new branch with the weird name.

Reply:
Discussed in private. The commit generating autoconf files must be the last one so we do need to create a new branch.